src/tools/rbd/action/MirrorPool.cc

```diff
@@ -794,8 +803,12 @@ int execute_info(const po::variables_map &vm) {
   if (formatter != nullptr) {
     formatter->open_object_section("mirror");
     formatter->dump_string("mode", mirror_mode_desc);
+    formatter->dump_string("data_pool_name", data_pool_name);
   } else {
     std::cout << "Mode: " << mirror_mode_desc << std::endl;
+    if (data_pool_name.length() != 0) {
```

Comment:
I've omitted the data pool output in the human-readable output if it's empty (indicating the default behavior), but included it in the JSON so that the JSON keys are consistent. Does this make sense?